src/main/scala/cromwell/binding/values/WdlMap.scala

```diff
@@ -0,0 +1,63 @@
+package cromwell.binding.values
+
+import cromwell.binding.types.{WdlAnyType, WdlMapType, WdlPrimitiveType, WdlType}
+import cromwell.util.{FileUtil, TryUtil}
+
+import scala.util.{Failure, Success, Try}
+
+object WdlMap {
+  def coerceMap(m: Map[_, _], wdlMapType: WdlMapType): WdlMap = {
+    val coerced = m map { case(k,v) => wdlMapType.keyType.coerceRawValue(k) -> wdlMapType.valueType.coerceRawValue(v) }
```

Comment:
why do you not have a space after the `,` in the case here but you do on line 16? I don't have a strong opinion either way except that it should be consistent.